components/report-project-dialog.tsx

```
@@ -0,0 +1,307 @@
+    setDescription('')
+    setError('')
+    onOpenChange(false)
+  }
+
+  return (
+    <Dialog open={open} onOpenChange={handleClose}>
+      <DialogContent className="max-w-2xl max-h-[90vh] overflow-y-auto">
+        <DialogHeader>
+          <DialogTitle>Report Project</DialogTitle>
+          <DialogDescription>
+            Report "{projectTitle}" for review. Please provide details about the issue.
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)